__tests__/integration.test.js

```diff
@@ -357,4 +357,28 @@ describe("/api/comments/:comment_id", () => {
     })
   })
 
-})
\ No newline at end of file
+})
+
+// api/users
+
+describe("/api/users", () => {
+
+  test("GET: Successful requests should return an array of all users with username, name and avatar_url properties", () => {
+    return request(app)
+    .get("/api/users")
+    .expect(200)
+    .then(({body})=>{
+      const {users} = body
+      expect(Array.isArray(users)).toBe(true)
```

Comment:
this line isn't strictly necessary because you're both checking the length of it and then iterating through it, which proves it's an array anyway

Reply:
Ahh I see. Thanks!